modules/shBidAdapter.md

```diff
@@ -49,6 +49,42 @@ Module that connects to ShowHeroes demand source to fetch bids.
                    }
                ]
            },
+           {
+               code: 'video',
+               mediaTypes: {
+                   video: {
+                       playerSize: [640, 480],
+                       context: 'outstream',
+                   }
+               },
+               bids: [
+                   {
+                       bidder: "showheroes-bs",
```

Comment:
During my testing I found that this bidder code causes issue with DFP as `-` is not a valid character for the _PATTERN:key_ macro ([see ad manager docs]([[https://support.google.com/admanager/answer/177381?hl=en]])).

I used the alias `showheroesBs` instead, which works.

Reply:
Is this something we want to fix / update?